internal/cli/atlas/metrics/disks/describe.go

```diff
@@ -59,14 +59,22 @@ var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurement
 {{ $name }}	{{ $unit }}	{{.Timestamp}}	{{if .Value }}	{{ .Value }}{{else}}	N/A {{end}}{{end}}{{end}}{{end}}
 `
 
-// mcli atlas metric(s) disk(s) describe <host:port> <name> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
-func DisksDescribeBuilder() *cobra.Command {
+// mcli atlas metric(s) disk(s) describe <host:port> <diskName> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DisksDescribeOpts{}
+	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <hostname:port> <name>",
+		Use:   "describe <hostname:port> <diskName>",
 		Short: "Describe disks measurements for a given host partition.",
```

Comment:
Can we use metrics instead of measurements, here and elsewhere?

Reply:
sure